Source/CarthageKit/Xcode.swift

```diff
@@ -967,7 +983,9 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 		}
 		.flatMap(.Concat) { platform, sdks -> SignalProducer<(Platform, [SDK]), CarthageError> in
 			let filterResult = sdkFilter(sdks: sdks, scheme: scheme, configuration: configuration, project: project)
-			return SignalProducer(result: filterResult.map { (platform, $0) })
+			return SignalProducer(result: filterResult.map { sdks in
+				return (platform, sdks)
```

Comment:
Nit: maybe just leave this as one line? `return SignalProducer(result: filterResult.map { (platform, $0) })`